tyk-pro/scripts/init.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+
+apk --no-cache add curl jq
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+/opt/scripts/wait-for-it.sh -t 120 $TYK_DASHBOARD_SVC.$TYK_POD_NAMESPACE.svc.cluster.local:$TYK_DB_LISTENPORT && \
```

Comment:
Do we need to ship wait-for-it with our helm charts. I think it's kinda overkill. 

This simple bash line should be able to check for the dashboard status: `while [[ $(kubectl get po -l app=dashboard-tyk-pro -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]; do sleep 10; done`

Reply:
We could, I normally use this script around all my projects. I don't have a problem keeping it or replacing it.

Reply:
can you replace it please? Needs to be tested properly.

Reply:
@zalbiraw @abhishiekgowtham Looks like this wasn't done, but the PR was merged nonetheless. I agree with Gowtham that it's better to have a simpler (easier to reason about) code if possible, so maybe do in a separate PR?

Reply:
@excieve i'll work on this and the alpine packages issue in a different PR.